man/systemd-system.conf.xml

```diff
@@ -162,6 +162,40 @@
         These settings have no effect if a hardware watchdog is not available.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RuntimeWatchdogPreSec=</varname></term>
+
+        <listitem><para>Configure the hardware watchdog device pre-timeout value.
+        Takes a timeout value in seconds (or in other time units similar to
+        <varname>RuntimeWatchdogSec=</varname>). A watchdog pre-timeout is a
+        notification generated by the watchdog before the watchdog reset might
+        occur in the event the watchdog has not been serviced. This notification
+        is handled by the kernel and can be configured to take an action (i.e.
+        generate a kernel panic) using the
+        <filename>/sys/class/watchdog/watchdog0/pretimeout_governor</filename>
+        sysfs file for your watchdog device. The available actions (or
+        governors) are listed in the
+        <filename>/sys/class/watchdog/watchdog0/pretimeout_available_governors</filename>
```

Comment:
maybe we should read these and log them when initializing the watchdog.

also, a patch for wdctl (from util-linux) might make sense to enumerate them